frontend/src/app/core/setup/globals/onboarding/tours/team_planners_tour.ts

```diff
@@ -40,8 +40,7 @@ export function teamPlannerTourSteps():OnboardingStep[] {
       showSkip: false,
       nextButton: { text: I18n.t('js.onboarding.buttons.next') },
       onNext() {
-        const backArrows = Array.from(document.getElementsByClassName('main-menu--arrow-left-to-project'));
-        const teamPlannerBackArrow = backArrows.find((backArrow) => (backArrow.nextElementSibling as HTMLElement).innerText === 'Team planners') as HTMLElement;
+        const teamPlannerBackArrow = document.querySelector('li[data-name="team_planner_view"] .main-menu--arrow-left-to-project') as HTMLElement|undefined;
```

Comment:
might be nice to add something like a `data-tour-selector` attribute for this

Reply:
I agree that it's nicer to have it explitly coupled to the tour. I'll keep the fix as minimal as possible for now though